em_model.h

```diff
@@ -4,14 +4,14 @@ class EM_model : public Rmodel {
     using Rmodel::Rmodel;
 
    public:
-    void update_timestamp(long double new_time_stamp, long double temperature) {
+    void update_timestamp(long double new_time_stamp, long double temperature, long double voltage, long double stress) {
         long double delta_t = new_time_stamp - current_time_stamp;
         current_time_stamp = new_time_stamp;
-        update(delta_t, temperature);
+        update(delta_t, temperature, voltage, stress);
     }
 
 
-    void update(long double delta_t, long double temperature) {
+    void update(long double delta_t, long double temperature, long double voltage, long double stress) {
```

Comment:
'voltage' and 'stress' are unused here. I've pushed an UNUSED macro for them to the 'nbti-model' branch.

Reply:
Think you will need to create a new merge request to include this change.